core/src/main/java/org/opensearch/sql/expression/operator/convert/TypeCastOperators.java

```diff
@@ -119,7 +123,8 @@ private static DefaultFunctionResolver castToInt() {
         impl(
             nullMissingHandling((v) -> new ExprIntegerValue(v.booleanValue() ? 1 : 0)),
             INTEGER,
-            BOOLEAN));
+            BOOLEAN),
```

Comment:
is this change mandatory to be a part of this PR
can/should we separate it only for an interface change ?

Reply:
it's part of the previous PR